### PR TITLE
Review branding changes

### PR DESCRIPTION
**Pull request by efraimbart**

### CODE REVIEW

File: build/ci/production-values.yaml [hunk 1]
```diff
@@ -100,31 +100,31 @@ ingress:
       path: /*
       serviceName: nginx
       port: 80
-    - host: "sefaria.org.il"
+    - host: "seforim.app"
```
Comment (efraimbart) [inline build/ci/production-values.yaml]: You missed the main one a [couple lines up](https://github.com/seforim-app/Sefaria-Project/pull/1/files#diff-115ac67d1c860a3be74fc70bb3ca1271da89f13b1617a40f5461dc8d8222352cR99).

These hosts are part of a configuration file and are indicating all the hosts that'll be used. By removing the `.il` from the hosts you'll be creating duplicates, so probably wise to either leave this file as is for now or just update the sefaria.org to seforim while leaving the `.il` as is.

File: build/ci/production-values.yaml [hunk 2]
```diff
@@ -170,21 +170,21 @@ localSettings:
   OFFLINE: "False"
   DOMAIN_LANGUAGE: {
     "https://www.sefaria.org": "english",
-    "https://www.sefaria.org.il": "hebrew",
+    "https://www.seforim.app": "hebrew",
```
Comment (efraimbart) [inline build/ci/production-values.yaml]: Similar to the above comment, you missed the actual sefaria.org one line up, and when you do update that this file will probably be somewhat broken (which won't actually hurt us currently) as you'll have two entries with the same key.

File: build/ci/production-values.yaml [hunk 3]
```diff
@@ -170,21 +170,21 @@ localSettings:
...
   }
   MONGO_HOST: "mongo"
   APSCHEDULER_NAME: "apscheduler-{{ .Values.deployEnv }}"
   SEARCH_ADMIN: "http://elasticsearch-data:9200"
   TURN_SERVER: ''
   USE_CLOUDFLARE: false
-  FRONT_END_URL:  "http://www.sefaria.org" 
+  FRONT_END_URL:  "http://www.seforim.app"
```
Comment (efraimbart) [inline build/ci/production-values.yaml:180]: We should probably get rid of `www.` across the board when we see it, unless we're considering supporting it at some point. For now it just redirects to without the `www.`.

File: data/tmp/siddur_schema.xml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <opml version="2.0">
   <head>
-    <ownerEmail>ephraim@sefaria.org</ownerEmail>
+    <ownerEmail>ephraim@seforim.app</ownerEmail>
```
Comment (efraimbart) [inline data/tmp/siddur_schema.xml]: Lol.

Not sure we should do this. Is this essentially taking ownership of something that's not ours?

**Comment by efraimbart** [comment]
Updating unnecessary configuration files now might make it more of a pain to merge upstream Sefaria changes in whenever.

File: locale/en/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -16,15 +16,15 @@ msgstr ""
...
 msgid "English"
 msgstr ""
 
-#: reader/templatetags/sefaria_tags.py:229 sefaria/settings.py:23
+#: reader/templatetags/Seforim.app_tags.py:229 Seforim.app/settings.py:23
 msgid "Hebrew"
 msgstr ""
 
-#: reader/templatetags/sefaria_tags.py:230
+#: reader/templatetags/Seforim.app_tags.py:230
```
Comment (efraimbart) [inline locale/en/LC_MESSAGES/django.po]: Bad. This relates to code, not just text.

File: locale/en/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -48,23 +48,23 @@ msgstr ""
 #: templates/static/linker.html:4 templates/static/newsletter.html:4
 #: templates/static/privacy-policy.html:4 templates/static/terms.html:4
 #: templates/static/visualizations.html:4
-msgid "Sefaria"
+msgid "Seforim.app"
 msgstr ""
 
 #: reader/views.py:627
-msgid "A source sheet created with Sefaria's Source Sheet Builder"
+msgid "A source sheet created with Seforim.app's Source Sheet Builder"
```
Comment (efraimbart) [inline locale/en/LC_MESSAGES/django.po:55]: Looks like these might be generated automatically and have to be update in their source, though I'm not sure how that affects the hebrew file. [Documentation](https://docs.djangoproject.com/en/1.11/topics/i18n/translation/).

File: scripts/cauldron_links.py [hunk 7]
```diff
@@ -72,7 +72,7 @@ def get_links(ref, server="", msg=False):
         return []
 
 
-def get_text(ref, lang="", versionTitle="", server="http://draft.sefaria.org"):
+def get_text(ref, lang="", versionTitle="", server="http://draft.seforim.app"):
```
Comment (efraimbart) [inline scripts/cauldron_links.py]: Will this break something if we don't have drafts.seforim.app set up?

File: sefaria/export.py [hunk 8]
```diff
@@ -388,7 +388,7 @@ def prepare_merged_text_for_export(title, lang=None):
         "title": title,
         "language": lang,
         "versionTitle": "merged",
-        "versionSource": "https://www.sefaria.org/%s" % title.replace(" ", "_"),
+        "versionSource": "https://www.seforim.app/%s" % title.replace(" ", "_"),
```
Comment (efraimbart) [inline sefaria/export.py]: Hmm.

Definitely ask me about this one.

File: sefaria/sitemap.py [hunk 9]
```diff
@@ -22,8 +22,8 @@ def chunks(l, n):
 class SefariaSiteMapGenerator(object):
 
     hostnames = {
-        'org': {'interfaceLang': 'en', 'hostname':'https://www.sefaria.org'},
-        'org.il': {'interfaceLang': 'he', 'hostname':'https://www.sefaria.org.il'},
+        'org': {'interfaceLang': 'en', 'hostname':'https://www.seforim.app'},
+        'org.il': {'interfaceLang': 'he', 'hostname':'https://www.seforim.app'},
```
Comment (efraimbart) [inline sefaria/sitemap.py]: Maybe we should get `.il`

File: static/css/s2.css [hunk 10]
```diff
@@ -91,8 +91,10 @@ a, a:hover {
 }
 @font-face {
   /*Greek*/
-  font-family: 'Cardo';
-    src: url('/static/fonts/Cardo-Regular.ttf');
+  font-family: 'Alegreya Sans';
+    src: url('https://fonts.googleapis.com/css2?family=Alegreya+Sans&display=swap);
+  /* font-family: 'Cardo';
+    src: url('/static/fonts/AlegreyaSans-Medium.ttf'); */
```
Comment (efraimbart) [inline static/css/s2.css]: If you downloaded it, use the downloaded one as it seems they are?

File: static/js/Footer.jsx [hunk 11]
```diff
@@ -34,50 +34,48 @@ class Footer extends Component {
...
     let next = this.state.isClient ? (encodeURIComponent(Sefaria.util.currentPath())) : "/" ; //try to make sure that a server render of this does not get some weird data in the url that then gets cached
     return (
       <footer id="footer" className="static sans-serif">
         <div id="footerInner">
             <Section en="About" he="אודות">
-                <Link href="/about" en="What is Sefaria?" he="מהי ספריא" />
-                <Link href="/help" en="Help" he="עזרה" />
-                <Link href="/team" en="Team" he="צוות" />
+                <Link href="/about" en="What is Seforim.app?" he="מהי Seforim.app" />
+                <Link href="/collections/7GMAmJgl" en="Help" he="עזרה" />
```
Comment (efraimbart) [inline static/js/Footer.jsx:45]: Intentional?

File: static/js/Footer.jsx [hunk 12]
```diff
@@ -89,17 +87,17 @@ class Footer extends Component {
               <div className="socialLinks">
                   <Link href={fbURL} en="Facebook" he="פייסבוק" blank={true}/>
                   &bull;
-                  <Link href="https://twitter.com/SefariaProject" en="Twitter" he="טוויטר" />
+                  <Link href="https://twitter.com/seforimapp" en="Twitter" he="טוויטר" />
```
Comment (efraimbart) [inline static/js/Footer.jsx]:
```suggestion
                  <Link href="https://twitter.com/SeforimApp" en="Twitter" he="טוויטר" />
```

File: static/js/NavSidebar.jsx [hunk 13]
```diff
@@ -570,7 +570,7 @@ const GetTheApp = () => (
 
 
 const StayConnected = () => {
-  const fbURL = Sefaria.interfaceLang == "hebrew" ? "https://www.facebook.com/sefaria.org.il" : "https://www.facebook.com/sefaria.org";
+  const fbURL = Sefaria.interfaceLang == "hebrew" ? "https://www.https://www.facebook.com/profile.php?id=100089914501501" : "https://www.https://www.facebook.com/profile.php?id=100089914501501";
```
Comment (efraimbart) [inline static/js/NavSidebar.jsx]:
```suggestion
  const fbURL = Sefaria.interfaceLang == "hebrew" ? "https://www.facebook.com/profile.php?id=100089914501501" : "https://www.facebook.com/profile.php?id=100089914501501";
```

File: static/js/StaticPages.jsx [hunk 14]
```diff
@@ -244,7 +244,7 @@ const ContestLandingPage = () => (
 
 
 const RambanLandingPage = () => {
-    const emailLink = <a target="_blank" href="mailto:hannah@sefaria.org?Subject=Ramban Sponsorship">hannah@sefaria.org</a>
+    const emailLink = <a target="_blank" href="mailto:hannah@seforim.app?Subject=Ramban Sponsorship">hannah@seforim.app</a>
```
Comment (efraimbart) [inline static/js/StaticPages.jsx]: I'm glad this organization is chock full of humans

File: static/js/StaticPages.jsx [hunk 15]
```diff
@@ -700,9 +700,9 @@ const EducatorsPage = () => (
 
     <ButtonRow white={true} enTitle="" heTitle="">
       { [
-          ["Online Educator Course", "מדריך למשתמש בספריא", "https://sefaria.typeform.com/to/tJVexqpG", "https://www.sefaria.org.il/sheets/361600?lang=he"],
-          ["Lesson Plans on Sefaria", "ספריא לתלמידים", "/collections/pedagogy-on-sefaria-exemplary-lesson-plans", "https://www.sefaria.org.il/collections/KGMlHrvA"],
-          ["Schedule A Workshop", "שאלות נפוצות", "https://sefaria.typeform.com/to/Pl3biam8", "https://www.sefaria.org.il/collections/%D7%A9%D7%90%D7%9C%D7%95%D7%AA-%D7%A0%D7%A4%D7%95%D7%A6%D7%95%D7%AA-%D7%91%D7%A1%D7%A4%D7%A8%D7%99%D7%90"]
+          ["Online Educator Course", "מדריך למשתמש בספריא", "https://sefaria.typeform.com/to/tJVexqpG", "https://www.seforim.app/sheets/361600?lang=he"],
+          ["Lesson Plans on Sefaria", "ספריא לתלמידים", "/collections/pedagogy-on-sefaria-exemplary-lesson-plans", "https://www.seforim.app/collections/KGMlHrvA"],
+          ["Schedule A Workshop", "שאלות נפוצות", "https://sefaria.typeform.com/to/Pl3biam8", "https://www.seforim.app/collections/%D7%A9%D7%90%D7%9C%D7%95%D7%AA-%D7%A0%D7%A4%D7%95%D7%A6%D7%95%D7%AA-%D7%91%D7%A1%D7%A4%D7%A8%D7%99%D7%90"]
```
Comment (efraimbart) [inline static/js/StaticPages.jsx:705]: These don't seem to work. What about the rest below?

File: static/js/StaticPages.jsx [hunk 16]
```diff
@@ -1511,14 +1511,14 @@ const DonatePage = () => (
                 <HeaderWithColorAccentBlockAndText
                     enTitle="Donor-Advised Fund"
                     heTitle=""
-                    enText="<p>Sefaria accepts donations from donor-advised funds; our <strong>EIN is 46-4406454</strong>. If you need additional information to make a DAF donation, please contact Sefaria’s development team at <a href='mailto:donate@sefaria.org'>donate@sefaria.org</a>.</p>"
+                    enText="<p>Sefaria accepts donations from donor-advised funds; our <strong>EIN is 46-4406454</strong>. If you need additional information to make a DAF donation, please contact Sefaria’s development team at <a href='mailto:donate@seforim.app'>donate@seforim.app</a>.</p>"
```
Comment (efraimbart) [inline static/js/StaticPages.jsx]: Using their EIN is probably super problematic!

File: static/js/StaticPages.jsx [hunk 17]
```diff
@@ -1511,14 +1511,14 @@ const DonatePage = () => (
...
                     heText=""
                     colorBar="#CCB479"
                 />,
                 <HeaderWithColorAccentBlockAndText
                     enTitle="Additional Ways to Give"
                     heTitle=""
-                    enText="<p>Sefaria also accepts donations via <strong>wire transfer</strong>, <strong>stock</strong>, and <strong>Cryptocurrency</strong>. For more information, please email  Sefaria’s development team at <a href='mailto:donate@sefaria.org'>donate@sefaria.org</a>. (For wire transfers, you can also <a href='https://sefaria.typeform.com/to/nxxyX62T'>click here</a> to get our account information)."
+                    enText="<p>Sefaria also accepts donations via <strong>wire transfer</strong>, <strong>stock</strong>, and <strong>Cryptocurrency</strong>. For more information, please email  Sefaria’s development team at <a href='mailto:donate@seforim.app'>donate@seforim.app</a>. (For wire transfers, you can also <a href='https://sefaria.typeform.com/to/nxxyX62T'>click here</a> to get our account information)."
```
Comment (efraimbart) [inline static/js/StaticPages.jsx]: Still got 'https://sefaria.typeform.com/to/nxxyX62T' in there.

File: static/js/StaticPages.jsx [hunk 18]
```diff
@@ -1537,7 +1537,7 @@ const DonatePage = () => (
                 A Message to Our Supporters
             </h1>
 
-              <p>Due to the ongoing pandemic, we encourage donors to use our <a href="https://sefaria.nationbuilder.com/supportsefaria">online donation page</a> or make a <a href="https://sefaria.typeform.com/to/nxxyX62T">wire transfer</a>, rather than sending a gift by mail. Supporters can also make an electronic gift of stock by emailing us at <a href='mailto:donate@sefaria.org'>donate@sefaria.org</a>. </p>
+              <p>Due to the ongoing pandemic, we encourage donors to use our <a href="https://sefaria.nationbuilder.com/supportsefaria">online donation page</a> or make a <a href="https://sefaria.typeform.com/to/nxxyX62T">wire transfer</a>, rather than sending a gift by mail. Supporters can also make an electronic gift of stock by emailing us at <a href='mailto:donate@seforim.app'>donate@seforim.app</a>. </p>
```
Comment (efraimbart) [inline static/js/StaticPages.jsx]: Still got "https://sefaria.nationbuilder.com/supportsefaria" in there.

File: static/js/TextsPage.jsx [hunk 19]
```diff
@@ -140,7 +140,7 @@ const Dedication = () => {
 
     function get_google_sheet_data() {
       const url =
-        'https://docs.google.com/spreadsheets/d/11c9Yw9FdfLnfbIWqvUztCt-QICW5790dfFGgXH7IB1k/edit#gid=0';
+        'https://docs.google.com/spreadsheets/d/1SwfFKSyGH4fkzUXZWJGbqdtNrpcxHtR7aM-AiAkRxJg/edit#gid=0';
```
Comment (efraimbart) [inline static/js/TextsPage.jsx:143]: Is this populated with whatever they're expecting?

File: static/js/sefaria/strings.js [hunk 20]
```diff
@@ -4,14 +4,14 @@ const Strings = {
     "Topics":"נושאים",
     "Community": "קהילה",
     "Donate": "תרומה",
-    "Sefaria Notifications": "הודעות בספריא",
+    "Seforim.app Notifications": "הודעות בספריא",
```
Comment (efraimbart) [inline static/js/sefaria/strings.js:7]: We should probably check whoever is using this to be certain that it's updated correctly.

**Comment by efraimbart** [comment]
Why all the changes to static/root?

File: templates/base.html [hunk 21]
```diff
@@ -18,7 +18,7 @@
 
     {% if SITE_SETTINGS.TORAH_SPECIFIC %}
     <link rel="alternate" href="https://www.sefaria.org{{ request.path }}" hreflang="en" />
-    <link rel="alternate" href="https://www.sefaria.org.il{{ request.path }}" hreflang="he" />
+    <link rel="alternate" href="https://www.seforim.app{{ request.path }}" hreflang="he" />
```
Comment (efraimbart) [inline templates/base.html:21]: Hmm.

File: templates/js/shavuot_map_2018_form_array.js [hunk 22]
```diff
@@ -1 +1 @@
...
...ry:": "ישראל"}, {"Sheet URL (optional):": "", "Timestamp": "10/05/2018 19:58:26", "Name of Institution (optional):": "", "I will be using Sefaria to prepare:": "ללמד על שבועות", "location": {"lat": 31.93263099999999, "lng": 35.023269}, "Town or city where you will be on Shavuot:": "חשמונאים", "Country:": "ישראל"}, {"Sheet URL (optional):": "", "Timestamp": "13/05/2018 18:01:14", "Name of Institution (optional):": "", "I will be using Sefaria to prepare:": "ללמוד על שבועות", "location": {"lat": 31.768319, "lng": 35.21371}, "Town or city where you will be on Shavuot:": "ירושלים", "Country:": "ישראל"}, {"Sheet URL (optional):": "", "Timestamp": "13/05/2018 19:21:12", "Name of Institution (optional):": "קהילת רענן", "I will be using Sefaria to prepare:": "ללמד על שבועות", "location": {"lat": 32.184781, "lng": 34.871326}, "Town or city where you will be on Shavuot:": "רעננה", "Country:": "ישראל"}, {"Sheet URL (optional):": "", "Timestamp": "14/05/2018 19:10:20", "Name of Institution (optional):": "", "I will be using Sefaria to prepare:": "ללמוד על שבועות", "location": {"lat": 32.801748, "lng": 35.383501}, "Town or city where you will be on Shavuot:": "מצפה נטופה", "Country:": "ישראל"}]
```
Comment (efraimbart) [inline templates/js/shavuot_map_2018_form_array.js]: Can't even begin to be sure what to do here.

File: templates/static/app.html [hunk 23]
```diff
@@ -317,17 +317,17 @@ <h3 class="mobileNewsletterHeader">
                 <div id="newsletterInput-subscribeMessage"></div>
             </div>
             <div class="mobileSocialLinks flexContainer">
-                <a href="https://www.facebook.com/sefaria.org" target="_blank">
+                <a href="https://www.https://www.facebook.com/profile.php?id=100089914501501" target="_blank">
```
Comment (efraimbart) [inline templates/static/app.html:320]:
```suggestion
                <a href="https://www.facebook.com/profile.php?id=100089914501501" target="_blank">
```

File: templates/static/coming-soon.html [hunk 24]
```diff
@@ -13,7 +13,7 @@ <h1>
         <div class="section">
             <p>We're still working on lots of great features for Sefaria. The button you just clicked is one of them! We left the button in to help us track how many people are really interested in this feature.</p>
 
-            <p>Sign up on our <a href="/connect">mailing list</a>, or follow us on <a href="http://www.facebook.com/sefaria.org" target="_blank">Facebook</a> or <a href="http://www.twitter.com/SefariaProject" target="_blank">Twitter</a> to stay in the loop about all our new features as they come out.</p>
+            <p>Sign up on our <a href="/connect">mailing list</a>, or follow us on <a href="http://www.https://www.facebook.com/profile.php?id=100089914501501" target="_blank">Facebook</a> or <a href="http://www.twitter.com/seforimapp" target="_blank">Twitter</a> to stay in the loop about all our new features as they come out.</p>
```
Comment (efraimbart) [inline templates/static/coming-soon.html:16]:
```suggestion
            <p>Sign up on our <a href="/connect">mailing list</a>, or follow us on <a href="https://www.facebook.com/profile.php?id=100089914501501" target="_blank">Facebook</a> or <a href="http://www.twitter.com/seforimapp" target="_blank">Twitter</a> to stay in the loop about all our new features as they come out.</p>
```

File: templates/static/connect.html [hunk 25]
```diff
@@ -30,15 +30,15 @@ <h1>
                 </div>
 
 
-            <a href="http://www.facebook.com/sefaria.org" target="_blank">
+            <a href="http://www.https://www.facebook.com/profile.php?id=100089914501501" target="_blank">
```
Comment (efraimbart) [inline templates/static/connect.html:33]:
```suggestion
            <a href="https://www.facebook.com/profile.php?id=100089914501501" target="_blank">
```

File: templates/static/home.html [hunk 26]
```diff
@@ -33,10 +33,10 @@
       "url": "https://{{ request.META.HTTP_HOST }}",
       "logo": "https://{{ request.META.HTTP_HOST }}{{ STATIC_PREFIX }}img/Sefaria_icon_360_square.jpg",
       "sameAs": [
-        "https://www.facebook.com/sefaria.org/",
-        "https://twitter.com/sefariaproject",
-        "https://www.youtube.com/user/SefariaProject",
-        "https://github.com/Sefaria"
+        "https://www.https://www.facebook.com/profile.php?id=100089914501501/",
```
Comment (efraimbart) [inline templates/static/home.html]:
```suggestion
        "https://www.facebook.com/profile.php?id=100089914501501/",
```

File: templates/static/home.html [hunk 27]
```diff
@@ -33,10 +33,10 @@
...
+        "https://twitter.com/seforimapp",
+        <!-- "https://www.youtube.com/user/SefariaProject", -->
```
Comment (efraimbart) [inline templates/static/home.html:38]: Cool if not broken!

File: templates/static/jobs.html [hunk 28]
```diff
@@ -101,12 +101,12 @@ <h2 class="anchorable">Israel Team</h2>
             <p>
                 <span class="int-en">
                     Sefaria does not currently have any open positions.
-                    Please follow us on <a _target="blank" href="http://www.facebook.com/sefaria.org" >Facebook</a>
+                    Please follow us on <a _target="blank" href="http://www.https://www.facebook.com/profile.php?id=100089914501501" >Facebook</a>
```
Comment (efraimbart) [inline templates/static/jobs.html]:
```suggestion
                    Please follow us on <a _target="blank" href="https://www.facebook.com/profile.php?id=100089914501501" >Facebook</a>
```

File: templates/static/jobs.html [hunk 29]
```diff
@@ -101,12 +101,12 @@ <h2 class="anchorable">Israel Team</h2>
...
                     to hear about our next openings.
                 </span>
                 <span class="int-he">
                     ספריא איננה מחפשת כעת עובדים חדשים.
-                    עקבו אחרינו ב<a _target="blank" href="http://www.facebook.com/sefaria.org" >פייסבוק</a>
+                    עקבו אחרינו ב<a _target="blank" href="http://www.https://www.facebook.com/profile.php?id=100089914501501" >פייסבוק</a>
```
Comment (efraimbart) [inline templates/static/jobs.html]:
```suggestion
                    עקבו אחרינו ב<a _target="blank" href="https://www.facebook.com/profile.php?id=100089914501501" >פייסבוק</a>
```

File: templates/static/mobile.html [hunk 30]
```diff
@@ -321,17 +321,17 @@ <h3 class="mobileNewsletterHeader">
                 <div id="newsletterInput-subscribeMessage"></div>
             </div>
             <div class="mobileSocialLinks flexContainer">
-                <a href="https://www.facebook.com/sefaria.org" target="_blank">
+                <a href="https://www.https://www.facebook.com/profile.php?id=100089914501501" target="_blank">
```
Comment (efraimbart) [inline templates/static/mobile.html]:
```suggestion
                <a href="https://www.facebook.com/profile.php?id=100089914501501" target="_blank">
```

**Review by efraimbart — COMMENTED** [review]
Just a quick review. Who am I kidding, it took an entire day.

File: build/ci/production-values.yaml (hunk 1 above)
Comment (rayistern) [reply build/ci/production-values.yaml]: I'll buy the il to make it easier

File: build/ci/production-values.yaml (hunk 3 above)
Comment (rayistern) [reply build/ci/production-values.yaml:180]: Don't want duplicate entries!

File: data/tmp/siddur_schema.xml (hunk 4 above)
Comment (rayistern) [reply data/tmp/siddur_schema.xml]: fixed

File: locale/en/LC_MESSAGES/django.po (hunk 5 above)
Comment (rayistern) [reply locale/en/LC_MESSAGES/django.po]: fixed

File: scripts/cauldron_links.py (hunk 7 above)
Comment (rayistern) [reply scripts/cauldron_links.py]: I don't know what this whole cauldron situation is

File: static/css/s2.css (hunk 10 above)
Comment (rayistern) [reply static/css/s2.css]: mistake. I actually updated it on the commented out line (of the old font). And this is probably what broke the site - I didn't put a quote at end of url!

File: static/js/Footer.jsx (hunk 11 above)
Comment (rayistern) [reply static/js/Footer.jsx:45]: Yes - check out the link

File: static/js/StaticPages.jsx (hunk 16 above)
Comment (rayistern) [reply static/js/StaticPages.jsx]: fixed

File: static/js/StaticPages.jsx (hunk 17 above)
Comment (rayistern) [reply static/js/StaticPages.jsx]: fixed

File: static/js/StaticPages.jsx (hunk 18 above)
Comment (rayistern) [reply static/js/StaticPages.jsx]: just removed the link

File: static/js/StaticPages.jsx (hunk 15 above)
Comment (rayistern) [reply static/js/StaticPages.jsx:705]: ye not supposed to work I'd prefer to change the ones below too

File: static/js/TextsPage.jsx (hunk 19 above)
Comment (rayistern) [reply static/js/TextsPage.jsx:143]: Now it is

File: templates/js/shavuot_map_2018_form_array.js (hunk 22 above)
Comment (rayistern) [reply templates/js/shavuot_map_2018_form_array.js]: I didn't do it

**Comment by rayistern** [comment]
> We should probably check whoever is using this to be certain that it's updated correctly. 

Reversed the changes

**Pull request merged**